index.html

```diff
@@ -236,13 +238,25 @@ <h4 class="heading-langS">Skills</h4>
                                                   <img src="./resources/getintouch.svg" alt="getintouch">
                                                 </button>
                                               </form>
+                                              <div class="bottom-right">
+                                                <img src="./resources/contact form-illustratoin-3-desktop.svg" alt="bottom-right">
+                                              </div>
                                             </div>
                                              <!-- contact form End-->
+                                            <div class="desktop-img-bottom">
+                                            <div class="bottomleft">
+                                              <img src="./resources/bottomleft.svg" alt="bottomleft">
+                                              </div>
+                                              <div class="bottom1">
+                                                <img src="./resources/bottom1.svg" alt="bottom1">
+                                              </div>
+                                              </div>
                                               <footer></footer>
                                               <section class="bottom">
```

Comment:
[OPTIONAL]
- I find no use for these `footer` tags unless you added the closing one by mistake, kindly fix it.